src/Model/Domain/TrackingResponse.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Copyright (C) 2013 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Model\Domain;
+
+use Mailgun\Model\ApiResponse;
+
+/**
+ * @author Artem Bondarenko <artem@uartema.com>
+ */
+final class TrackingResponse implements ApiResponse
+{
+    private $click;
+    private $open;
+    private $unsubscribe;
+
+    public static function create(array $data): ?self
+    {
+        if (!isset($data['tracking'])) {
+            return null;
+        }
+
+        $trackingSettings = $data['tracking'];
+
+        $model = new self();
+        $model->click = ClickTracking::create($trackingSettings['click'] ?? []);
+        $model->open = OpenTracking::create($trackingSettings['click'] ?? []);
```

Comment:
It seems to me that this property should be checking the `open` key of the array instead of the `click` one. Can you please make sure this is fixed?

Also, as part of the automated tests, when simulating the response, please try to set one of the properties as `true` and the other one as `false` so you can confirm they are not mapped wrongly when copying and pasting.

Please note the last comment is a suggestion for future cases too, so you can confirm the tests are providing more accurate results, given is hard getting rid of a bad habit such as copy-paste.